src/coprocessor/dag/rpn_expr/mod.rs

```diff
@@ -65,6 +65,15 @@ fn plus_mapper(lhs_is_unsigned: bool, rhs_is_unsigned: bool) -> Box<dyn RpnFunct
     }
 }
 
+fn mod_mapper(lhs_is_unsigned: bool, rhs_is_unsigned: bool) -> Box<dyn RpnFunction> {
+    match (lhs_is_unsigned, rhs_is_unsigned) {
+        (false, false) => Box::new(RpnFnArithmetic::<IntIntMod>::new()),
```

Comment:
Maybe we can discover some higher ordered generics to deal with these IntInt, IntUint, UintInt, UintUint later, to reduce code duplicates :) We will have many functions that is different in signed / unsigned integers. Would be too verbose if we need a standalone function for each of them.